lte/gateway/c/oai/tasks/s1ap/s1ap_mme.c

```diff
@@ -112,12 +112,16 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
   state                          = get_s1ap_state(false);
   AssertFatal(state != NULL, "failed to retrieve s1ap state (was null)");
 
+  bool is_state_same = false;
```

Comment:
minor nit: I feel `has_state_changed` is a better name for this, and it should reduce the number of the value being set to true

Reply:
@ardzoht I want to have the default behavior to sync the state and explicitly call out the ones that does not modify the state (at all or most of the time). This will be more robust against new additions to the code base (e.g., s1 mobility).